receiver/kafkametricsreceiver/broker_scraper.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkametricsreceiver/internal/metadata"
 )
 
+type saramaMetrics map[string]map[string]interface{}
```

Comment:
Could you add a comment here what is expected in the map?

```
// saramaMetrics is a map of metric name to tags
```

Reply:
added